src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -14,12 +14,68 @@
 
 package com.google.sps.servlets;
 
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Collection;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 /**
  * Servlet that fetch from Youtube Server.
  */
-@WebServlet("/userLogIn")
+@WebServlet("/YouTubeComments")
 public class YoutubeServlet extends HttpServlet {
+  private static final String DEVELOPER_KEY = "OUR_API_KEY"; //TODO: obtain actual API Key
+
+  private static final String APPLICATION_NAME = "SAY";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
```

Comment:
nit: order of variable names: APPLICATION_NAME ->  DEVELOPER_KEY, JSON_FACTORY

Reply:
I would agree except this is the order in which they were in the sample code, is there an explicit ordering to follow?

Reply:
This mentions import orders, but should generally apply:
https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents
Use ASCII ordering when they are the same type, but generally the ordering is not that important when they are all constants.
If there are different modifiers though, please follow:
https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Reply:
For the APPLICATION_NAME and DEVELOPER_KEY, they are the same type and maybe use ASCII ordering?

Reply:
hmmmm unresolved conversation?